src/routes/jira/jira-router.ts

```diff
@@ -4,13 +4,17 @@ import { JiraSyncPost } from "./jira-sync-post";
 import { JiraAtlassianConnectGet } from "./jira-atlassian-connect-get";
 import { JiraEventsRouter } from "./events/jira-events-router";
 import { JiraSelectVersionRouter } from "./server/jira-select-version-router";
-import { JiraContextJwtTokenMiddleware } from "middleware/jira-jwt-middleware";
+import { JiraContextJwtTokenMiddleware, JiraJwtTokenMiddleware } from "middleware/jira-jwt-middleware";
 import { JiraServerUrlRouter } from "routes/jira/server/jira-server-url-router";
 import { JiraAppCreationRouter } from "./server/jira-app-creation-router";
+import { csrfMiddleware } from "middleware/csrf-middleware";
+import { JiraManualAppCreationGet } from "routes/jira/server/jira-manual-app-creation-get";
 
 export const JiraRouter = Router();
 
 JiraRouter.get("/atlassian-connect.json", JiraAtlassianConnectGet);
+JiraRouter.get("/manual-app-creation", csrfMiddleware, JiraJwtTokenMiddleware, JiraManualAppCreationGet);
```

Comment:
we need to have a chat around all these new routes as they're related but not associated through routing and using a few anti-patterns

Reply:
We can setup a meeting for this I suppose? Do we hold this PR until then?

Reply:
yeah, maybe just a quick one over huddle with whoever is interested in this in the ghe channel?

Reply:
The URL has been updated this one only...